src/it/scala/io/iohk/ethereum/sync/RegularSyncItSpec.scala

```diff
@@ -0,0 +1,71 @@
+package io.iohk.ethereum.sync
+
+import io.iohk.ethereum.FlatSpecBase
+import io.iohk.ethereum.sync.util.RegularSyncItSpecUtils.FakePeer
+import io.iohk.ethereum.sync.util.SyncCommonItSpec._
+import monix.execution.Scheduler
+import org.scalatest.BeforeAndAfter
+import org.scalatest.matchers.should.Matchers
+
+import scala.concurrent.duration._
+
+class RegularSyncItSpec extends FlatSpecBase with Matchers with BeforeAndAfter {
+  implicit val testScheduler = Scheduler.fixedPool("test", 16)
+
+  it should "should sync blockchain with same best block" in customTestCaseResourceM(FakePeer.start2FakePeersRes()) {
```

Comment:
It is a small detail but this `it should "should` will produce output like `should should sync blockchain with same best block` :)

Reply:
Done